nuget/readme.md

```diff
@@ -48,6 +48,7 @@ C#/WinRT behavior can be customized with these project properties:
 | CsWinRTRcwFactoryFallbackGeneratorForceOptIn | true \| *false | Forces the RCW factory fallback generator to be enabled (it only runs on .exe projects by default)  |
 | CsWinRTRcwFactoryFallbackGeneratorForceOptOut | true \| *false | Forces the RCW factory fallback generator to be disabled (overrides "ForceOptIn" as well)  |
 | CsWinRTMergeReferencedActivationFactories | true \| *false | Makes the native `DllGetActivationFactory` exported function for AOT scenarios also forward the activation call to all referenced WinRT components, allowing them to all be merged into a single executable or shared library |
+| CsWinRTUseEnvironmentalTools | true \| *false | Makes the invocation of MSVC to produce a stub .exe (for Native AOT scenarios) rely on environmental tools, rather than looking up the MSVC install path |
```

Comment:
Any reason not to automatically do this as a fallback or so?  Will someone run into this when building from VS or so?

Reply:
I'm not sure how to detect whether we're inside a VS Developer Command Prompt. This ensures that things will still build correctly when just doing `dotnet build` too, or `dotnet msbuild`, from a normal terminal. The option matches what Native AOT also has (they have "IlcUseEnvironmentalTools").

Reply:
I am trying to understand which scenarios when someone will need to set this property.  Is this when they are using the VS command prompt?  When they are building in VS?  or just terminal?

Reply:
Yeah, if you build from VS command prompt you'll want to set the property, to ensure that the build environment is the same one that normal VS would use. You wouldn't set this if you want to build from a normal terminal window.

Reply:
Thoughts on checking for VSCMD_VER to detect VS developer command prompt?

https://learn.microsoft.com/en-us/visualstudio/ide/reference/command-prompt-powershell?view=vs-2022#developer-command-prompt

Reply:
Updated to use this to set the default! 👍